test/e2e/features/ux.feature

```diff
@@ -1,23 +1,29 @@
-@ux
+@ux 
 Feature: UX Test
 
     Tets CRC usage with ux components. 
         * Handle CRC install/ uninstall operations based on installer distribution
         * Manage CRC through the tray
-
-    @darwin
+    
+    @install @darwin
     Scenario: Install CRC 
         Given an environment where CRC is not installed
         When install CRC from installer
         Then CRC is installed
-
-    @darwin
+		
+	@install @windows
+    Scenario: Install CRC 
+        Given an environment where CRC is not installed
+        When install CRC from installer
+        Then reboot is required
```

Comment:
I see what you did here...

How is the reboot and continuation actually handled?

Reply:
Basically I split the test suite based on annotations...so the ux on windows requires two suites: first @ux && @install ..this will run the installer finalizing with the reboot, second suite runs @ux && ~@install which run all other ux scenarios. 

The other option I had in mind but was not possible for the time being was running the suite from outside.. (so each step would be wrapped and executed remotely) so then you can control the reboot flow...